pypeit/par/pypeitpar.py

```diff
@@ -1800,7 +1800,7 @@ def __init__(self, caldir=None, setup=None, trim=None, badpix=None, biasframe=No
 
         # Fill out parameter specifications.  Only the values that are
         # *not* None (i.e., the ones that are defined) need to be set
-        defaults['caldir'] = 'MF'
+        defaults['caldir'] = 'Masters'
```

Comment:
I suspect this will break our make `Cooked` script, which is fine.
Just a head's up.

Reply:
Yeah, I need to remake Cooked and run the cooked tests to make sure that I haven't missed somewhere `MF` is hard-coded.